core/src/main/res/layout/activity_dev_menu.xml

```diff
@@ -25,46 +25,101 @@
             android:layout_height="wrap_content"
-            android:text="@string/menu_title"
-            android:textColor="@android:color/black"
-            android:background="@android:color/darker_gray"
-            android:gravity="center"
-            android:padding="10dp"
-            android:layout_marginBottom="20dp" />
-
-        <Button
-            android:id="@+id/env"
+            android:orientation="vertical"
+            android:paddingLeft="15dp"
+            android:paddingRight="15dp"
+            android:paddingBottom="15dp">
+
+            <Button
+                android:id="@+id/env_button"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:text="@string/env_button_title"
+                android:textAllCaps="false" />
+
+            <Button
+                android:id="@+id/device_button"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:text="@string/device_button_title"
+                android:textAllCaps="false" />
+
+            <Button
+                android:id="@+id/logs_button"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:text="@string/logs_button_title"
+                android:textAllCaps="false" />
+
+            <Button
+                android:id="@+id/file_issue_button"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:text="@string/file_issue_button_title"
+                android:textAllCaps="false" />
+        </LinearLayout>
+
+        <LinearLayout
+            android:id="@+id/env_layout"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:text="@string/env_button_title"
-            android:textAllCaps="false" />
+            android:orientation="vertical"
+            android:visibility="gone" />
 
-        <Button
-            android:id="@+id/device"
+        <LinearLayout
+            android:id="@+id/device_layout"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:text="@string/device_button_title"
-            android:textAllCaps="false" />
+            android:orientation="vertical"
+            android:visibility="gone" />
 
-        <Button
-            android:id="@+id/logs"
+        <LinearLayout
+            android:id="@+id/logs_layout"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:text="@string/logs_button_title"
-            android:textAllCaps="false" />
+            android:orientation="vertical"
+            android:visibility="gone" />
 
-        <Button
-            android:id="@+id/file_issue"
+        <LinearLayout
```

Comment:
It's best to avoid nesting of layout elements like this (see [discussion](https://developer.android.com/training/improving-layouts/optimizing-layout).)

Three isn't the end of the world. But checkout the `ConstraintLayout` and see if you could fit all of your individual items in it?